### DISCUSSION TITLE
Few suggestions for Vorta 0.7.1

### DISCUSSION THREAD

**Discussion by ghost** [borgbase/vorta#819] [Ideas]
1. Start Backup button becomes disabled after the backup starts. It should either read "In progress ..." or "Backing up ..."

2. Progress bar would be nice to have.

3. I cannot execute 2+ backup towards same repo. Is that a limitation?

4. Last file backed up still shows after the backup finishes.

**Comment by m3nu** [comment]
1 maybe, 2 is a duplicate.

3: you can use the same repo in multiple profiles

4: true, this is to preserve potential error messages. Could probably be improved by distinguishing between file logs and error logs.
